### PR TITLE
Lotus version 0.7.2

### PR DESCRIPTION
**Pull request by arajasek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Lotus changelog
 
+# 0.7.2 / 2020-09-23
+
+This optional release of Lotus introduces a major refactor around how a Lotus node interacts with code from the specs-actors repo. We now use interfaces to read the state of actors, which is required to be able to reason about different versions of actors code at the same time.
+
+Additionally, this release introduces various
```
Comment (Stebalien) [inline CHANGELOG.md:7]: ...

Comment (Stebalien) [reply CHANGELOG.md:7]: Maybe a "test carefully" warning?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- Validate chain sync response indices when fetching messages (https://github.com/filecoin-project/lotus/pull/3939)
+- Add height diff to sync wait (https://github.com/filecoin-project/lotus/pull/3926)
+- Replace Requires with Wants (https://github.com/filecoin-project/lotus/pull/3898)
+- Update state diffing for market actor (https://github.com/filecoin-project/lotus/pull/3889)
+- Parallel fetch for sync (https://github.com/filecoin-project/lotus/pull/3887)
+- Fix SectorState (https://github.com/filecoin-project/lotus/pull/3881)
+
+#### User Experience
+
+- Fix out-of-bounds when loading all sector infos (https://github.com/filecoin-project/lotus/pull/3976
```
Comment (Stebalien) [inline CHANGELOG.md:24]: Wasn't a regression (was an issue with the refactor PR), do we mention these?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+- Add basic deal stats api server for spacerace slingshot (https://github.com/filecoin-project/lotus/pull/3963)
+- When doing `sectors update-state`, show a list of existing states if user inputs an invalid one (https://github.com/filecoin-project/lotus/pull/3944)
+- Fix `lotus-miner storage find` error (https://github.com/filecoin-project/lotus/pull/3927)
+- Log shutdown method for lotus daemon and miner (https://github.com/filecoin-project/lotus/pull/3925)
+- Update build and setup instruction link (https://github.com/filecoin-project/lotus/pull/3919)
+- Add an option to hide sectors in Removed from `sectors list` (https://github.com/filecoin-project/lotus/pull/3903)
```
Comment (Stebalien) [inline CHANGELOG.md:30]: What does this mean?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+
+#### Testing and validation
+
+- Add init.State#Remove() for testing (https://github.com/filecoin-project/lotus/pull/3971)
```
Comment (Stebalien) [inline CHANGELOG.md:34]: Not sure if this is relevant (was effectively part of the refactor)

**Review by Stebalien — APPROVED** [review]
(do we just mention every PR?)

LGTM module the dangling sentence.

File: CHANGELOG.md (hunk 1 above)
Comment (arajasek) [reply CHANGELOG.md:7]: i obviously meant to leave that sentence trailing, it's

File: CHANGELOG.md (hunk 3 above)
Comment (magik6k) [reply CHANGELOG.md:30]: I'd say
```suggestion
- Hide Removed sectors from `lotus-miner sectors list` by default (https://github.com/filecoin-project/lotus/pull/3903)
```

Comment (arajasek) [reply CHANGELOG.md:30]: fixing

File: CHANGELOG.md (hunk 2 above)
Comment (arajasek) [reply CHANGELOG.md:24]: removing

**Comment by arajasek** [comment]
> (do we just mention every PR?)

generally, yeah

**Pull request merged**